meow.py

```diff
@@ -422,34 +422,42 @@ def generate_popularity_report(self):
 
             Args-None
             """
+            self.screen.clear()
+            self.screen.border(0)
             # initial method setup
-            li_lib = deserialize("line_item.txt")
-            orders_lib = deserialize("orders.txt")
-            products_lib = deserialize("products.txt")
+            li_lib = deserialize("data/line_items.txt")
+            orders_lib = deserialize("data/orders.txt")
+            products_lib = deserialize("data/products.txt")
 
             ########## BUILD POPULARITY DICT ##########
             # create dictionary with keys: product ids and values: dict of purchase info
+            total_customers = set()
```

Comment:
nice! looks like you made the total number of customers reflect the number of _unique_ customers. cool!